src/components/Launch/LaunchWorkflowForm/useLaunchWorkflowFormState.ts

```diff
@@ -240,28 +253,43 @@ export function useLaunchWorkflowFormState({
     useEffect(
         () => {
             const parsedInputs =
-                launchPlan && workflow.hasLoaded
-                    ? getInputs(workflow.value, launchPlan)
+                selectedLaunchPlan && workflow.hasLoaded
+                    ? getInputs(workflow.value, selectedLaunchPlan.data)
```

Comment:
Can `selectedLaunchPlan.data` also change, and should you trigger the `useEffect` on that too? Just checking.

Reply:
Hmm. So technically, according to the Rules of Hooks, using that value means I should include it in the dependency array.

Practically, the data in the selector options is a const value that does not change unless you regenerate all of the items. In that case, the outer object would also change.

But I plan on enabling the extra lint rule in the future to ensure all captured values used in the effect are listed in the dependencies array. So I'll update it here.

Good catch!